pkg/proxy/volume.go

```diff
@@ -18,7 +19,28 @@ func (p *Proxy) VolumeGet(ctx context.Context, req *rpc.ProxyEngineRequest) (res
 		Volume: &eptypes.Volume{},
 	}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated